### ISSUE TITLE
I tried to install it and it doesn't work

### ISSUE THREAD

**Issue by maaagdonals** [deezer/spleeter#172]
I tried to do it as stated a lot of times and I'm getting frustrated. Could anyone give some help?
(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>conda config --add channels conda-forge
Warning: 'conda-forge' already in 'channels' list, moving to the top

(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>
(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>conda install spleeter
Collecting package metadata (current_repodata.json): done
Solving environment: done

All requested packages already installed.

(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>
(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>spleeter separate -h
Fatal error in launcher: Unable to create process using '"d:\bld\spleeter_1574775894867\_h_env\python.exe"  "C:\Users\User\Anaconda3\Scripts\spleeter.exe" separate -h'

(base) C:\Users\User\Desktop\AI's\spleeter-feedstock-master>spleeter separate -i "C:\Users\User\Desktop\Music\03 Holiday (feat.mp3" -o audio_output -p spleeter:4stems
Fatal error in launcher: Unable to create process using '"d:\bld\spleeter_1574775894867\_h_env\python.exe"  "C:\Users\User\Anaconda3\Scripts\spleeter.exe" separate -i "C:\Users\User\Desktop\Music\03 Holiday (feat.mp3" -o audio_output -p spleeter:4stems'

**Comment by ngoname** [comment]
It's weird. This ran fine for me a couple of weeks ago but this whole D: drive mess is coming back to haunt me.  I stopped doing PIP because it was not completing now that I'm coming back to the original method (which used to work before I abandoned it because I was running into the other reported D: issue), I'm running into this again.  If I'm desperate I could rename my drives but it just strikes me as weird.  If we could know which file to change the directory ourselves that could help.... or just point to the same directory we're running these scripts on by default...
